doc/source/tutorial/stats/sampling_nrou.rst

```diff
@@ -0,0 +1,225 @@
+
+.. _sampling-nrou:
+
+Naive Ratio-Of-Uniforms (NROU) Method
+=====================================
+
+.. currentmodule:: scipy.stats
+
+* Required: PDF
+* Optional: mode, center, bounding rectangle for acceptance region
+* Speed:
+
+  * Set-up: fast (if the bounding rectangle is known), otherwise slow
+  * Sampling: moderate
+
+NROU is an implementation of the (generalized) ratio-of-uniforms method which
```

Comment:
```suggestion
NROU is an special case of the (generalized) ratio-of-uniforms method that
```
Is that what is meant?

Reply:
no, NROU is the usual ROU method if r==1, otherwise it is called generalized ROU